src/cdk-lib/bedrock/models.ts

```diff
@@ -57,6 +57,14 @@ export class BedrockFoundationModel {
     'amazon.titan-text-express-v1',
     { supportsAgents: true },
   );
+  public static readonly ANTHROPIC_CLAUDE_3_5_SONNET_V1_0 = new BedrockFoundationModel(
+    'anthropic.claude-3-5-sonnet-20240620-v1:0',
+    { supportsAgents: true },
+  );
+  public static readonly ANTHROPIC_CLAUDE_OPUS_V1_0 = new BedrockFoundationModel(
```

Comment:
Maybe we should deprecate names like `ANTHROPIC_CLAUDE_SONNET_V1_0` and rename it to `ANTHROPIC_CLAUDE_3_SONNET_V1_0` for consistency.

Reply:
Agree, although we should probably have a separate PR